xc/device.mli

```diff
@@ -21,6 +21,20 @@ exception Device_not_found
 
 exception Cdrom
 
+module Profile: sig
+	type t = Qemu_trad | Qemu_upstream_compat | Qemu_upstream
+	val fallback : t
+	module Name: sig
+		val qemu_trad: string
+		val qemu_upstream_compat: string
+		val qemu_upstream: string
+		val all: string list
+	end
+	val wrapper_of: t -> string
```

Comment:
What is the difference between `wrapper_of` and `string_of`?

Reply:
`wrapper_of` returns the python wrapper to execute the device model profile, and `string_of` returns the string corresponding to the device model profile. The latter is also equivalent to the acceptable strings used in VM.platform.device-model.